bundles/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/internal/config/PlugwiseRelayConfig.java

```diff
@@ -36,7 +38,8 @@ public enum PowerStateChanging {
     }
 
     private String macAddress = "";
-    private String powerStateChanging = upperUnderscoreToLowerCamel(COMMAND_SWITCHING.name());
+    private String powerStateChanging = Objects
+            .requireNonNull(StringUtils.capitalizeByUnderscore(COMMAND_SWITCHING.name()));
```

Comment:
Why is `upperUnderscoreToLowerCamel` still in PlugwiseUtils if you now use `StringUtils`?

Reply:
Because this method is not available in core StringUtils. It was not included as this is the only binding using it. I only included methods that are shared across multiple bindings.

Reply:
`upperUnderscoreToLowerCamel` should be removed from `PlugwiseUtils` if you now use the `StringUtils` logic.

Reply:
`upperUnderscoreToLowerCamel` is the opposit to `lowerCamelToUpperUnderscore` Only the latter is available in core. 

So per previous review comment, i moved `upperUnderscoreToLowerCamel` from this bindings StringUtils (not core) to this bindings PlugwiseUtils (as how it was before) 

So what do you want me to do now?

Reply:
Remove `PlugwiseUtils.upperUnderscoreToLowerCamel` because it is unused with your changes.

Reply:
Ah, sorry, offcourse, just didn't understand. Fixed now.